neon_audio/utils.py

```diff
@@ -62,9 +63,20 @@ def _plugin_to_package(plugin: str) -> str:
     }
     return known_plugins.get(plugin) or plugin
 
+def build_extra_dependency_list(config: Configuration, module: str, additional: List[str] = []) -> str:
+    extra_dependencies = config.get("extra_dependencies", [])
+    dependencies = additional + extra_dependencies.get("global", []) + extra_dependencies.get(module, [])
+    if config["tts"]["module"]:
+        dependencies.append(_plugin_to_package(config["tts"]["module"]))
+
+    if config["tts"].get("package_spec"):
+        dependencies.append(config["tts"].get("package_spec"))
+
+    return dependencies
 
 def install_tts_plugin(plugin: str) -> bool:
     """
+    @deprecated
```

Comment:
We have a decorator that will warn on calls to deprecated methods, so this can be decorated with:
```python
@deprecated("Replaced by `neon_utils.packaging_utils.install_packages_from_pip`, "2.0.0")
```